backend/package.json

```diff
@@ -27,7 +27,7 @@
     "lodash": "^4.17.4",
     "memcache": "0.3.0",
     "mocha": "1.20.1",
-    "moment": "2.11.2",
+    "moment": "2.21.0",
```

Comment:
Apparently there is some vulnerability issue with the previous version we have. So updated it here